palak/src/pages/ForgotPsswrd.jsx

```diff
@@ -0,0 +1,72 @@
+import { z } from "zod";
+import { useState } from "react";
+import { Label } from "../components/Label";
+const Forgotpsswrd = ({ handleClick }) => {
+  const [formData, setFormData] = useState({ email: "" });
+  const [error, setError] = useState({});
+  const schema = z.object({
+    email: z.string().email("Must be a valid email address"),
+  });
+
+  const handleChange = (e) => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const result = schema.safeParse({
+      ...formData,
+    });
+    if (!result.success) {
+      const formattedErrors = result.error.format();
+      setError(formattedErrors);
+    } else {
+      setError({});
+    }
+  };
+  return (
+    <div>
+      <p className="pt-2 px-6 text-sm text-center">
+        Enter email address associated with your account and you will receive an
+        email to reset your password.
+      </p>
+
+      <div className="p-6 pt-6">
+        <form action="" onSubmit={handleSubmit}>
+          <div className="flex flex-col space-y-1 ">
+            <Label type="Email" />
+            <input
+              type="email"
+              name="email"
+              value={formData.email}
+              onChange={handleChange}
+              className="border-gray-300 border-[1px] flex h-10 w-full rounded-md px-3 text-sm "
+              placeholder="Enter Email ID ..."
+            />
+            {error.email && (
+              <p className="text-xs font-semibold text-red-600">
+                {error.email._errors[0]}
+              </p>
+            )}
+          </div>
+          <div className="flex items-center justify-center ">
+            <button
+              className="bg-blue-900 w-full h-11 mt-10 text-white text-sm font-medium rounded-lg max-w-sm"
```

Comment:
Check if the button with such css is being used at multiple places. If yes try to make this a reusable component.